api/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "dependencies": {
+    "@redwoodjs-stripe/api": "0.1.0-rc.9.monorepo-test.6",
```

Comment:
This will be something like `0.1.0-rc.10` once https://github.com/chrisvdm/redwoodjs-stripe/pull/83 has been merged and a new release is done.